src/client/ClientSnapRealm.h

```diff
@@ -35,7 +36,8 @@ struct SnapRealm {
 
   explicit SnapRealm(inodeno_t i) :
     ino(i), nref(0), created(0), seq(0),
-    pparent(NULL), last_modified(utime_t()), change_attr(0) { }
+    pparent(NULL), last_modified(utime_t()), change_attr(0),
+    is_snapdir_visible(true) { }
```

Comment:
revert

Reply:
(I also support moving the initialization for these other members in a separate commit.)